code/modules/mob/living/carbon/species.dm

```diff
@@ -157,6 +157,13 @@ var/global/list/whitelisted_species = list("Human")
 		myhuman = null
 	..()
 
+/datum/species/proc/gib(var/mob/living/carbon/human/H as mob)
```

Comment:
The last `as mob`, and we'll be good to go.

Reply:
My ctrl+F wasn't strong enough I guess.